ui/src/app/applications/components/application-pod-view/pod-tooltip.tsx

```diff
@@ -49,3 +57,17 @@ export const PodTooltip = (props: {pod: Pod}) => {
         </div>
     );
 };
+
+function formatPodMetric(name: string, value: string) {
+    const numericValue = parseInt(value, 10);
+
+    switch (name) {
+        case 'Requests (CPU)':
+            return `${numericValue}m`;
+        case 'Requests (MEM)': {
+            return formatSize(numericValue / 1000);
```

Comment:
Should this be / 1024?

Reply:
No, 1000. its just converting the milli-bytes coming from backend to bytes. which then be formatted with 1024.